testing/README.md

```diff
@@ -35,7 +35,7 @@ Install the testing library to your local repository
 
 2. Go to the Java file you want to test
 
-3. In IntelliJ, On the class name you would test, click`⌥⏎ `, select **Create Test**
+3. On the class name you would test, click`⌥⏎ `, select **Create Test**
```

Comment:
That is not what I meant by "Do not assume the user is using IDEA". You can't just remove its name and pretend there is nothing to do with it.

By clicking "Create Test", the IDE just create a new class and put some boilerplate code in it. We can do that manually without an IDE.